scripts/test

```diff
@@ -37,7 +43,12 @@ copy_logs(){
 run_tests()
 {
     pushd ./tests/validation
-    tox || [ `uname` == "Linux" ] && copy_logs && exit 1
+    if [ "${CATTLE_TEST_PARALLEL_THREADS}" -gt 1 ] 
```

Comment:
Oh, that reminds me: add `-v` to the tox command and it will print the name of each test as it runs. Ie: `tox -- -v -n "${CATTLE_TEST_PARALLEL_THREADS} ...`